### PR TITLE
Fix building and booting on edo

### PR DESCRIPTION
**Pull request by hellobbn**

### CODE REVIEW

**Comment by MarijnS95** [comment]
There's `[DNM]` in one of the commits?

**Comment by MarijnS95** [comment]
And half the commits miss a body, please describe each commit instead of forcing everyone to do research on their own.

**Comment by hellobbn** [comment]
> There's `[DNM]` in one of the commits?
this is a draft. The vendor.sensors.qesdk.rc is not needed on edo, however, the oem provided by sodp contains this file, which should be removed

**Comment by hellobbn** [comment]
> And half the commits miss a body, please describe each commit instead of forcing everyone to do research on their own.

done

**Comment by MarijnS95** [comment]
> this is a draft

That is not **at all** clear _from the title of this commit_. Click "mark as draft" on the top-right.

**Comment by hellobbn** [comment]
> > this is a draft
> 
> That is not **at all** clear _from the title of this commit_. Click "mark as draft" on the top-right.
> 
> https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests

will this be fixed on software binary side?

**Comment by hellobbn** [comment]
I have removed the DNM commit

File: rootdir/vendor/etc/seccomp_policy/atfwd@2.0.policy [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2020 Qualcomm Technologies, Inc.
+# All Rights Reserved.
```
Comment (jerpelea) [inline rootdir/vendor/etc/seccomp_policy/atfwd@2.0.policy:2]: please remove this patch

Comment (hellobbn) [reply rootdir/vendor/etc/seccomp_policy/atfwd@2.0.policy]: done

Comment (jerpelea) [reply rootdir/vendor/etc/seccomp_policy/atfwd@2.0.policy]: please remove the patch !

Comment (hellobbn) [reply rootdir/vendor/etc/seccomp_policy/atfwd@2.0.policy]: I have removed the patch, can you check https://github.com/sonyxperiadev/device-sony-common/pull/903/commits again?

**Review by jerpelea — APPROVED** [review]

**Pull request merged**